backpack/core/derivatives/nll_base.py

```diff
@@ -0,0 +1,132 @@
+"""Partial derivative bases for NLL losses."""
+from math import sqrt
+from typing import List, Tuple
+
+from torch import Size, Tensor, enable_grad, stack
+from torch.autograd import Variable, grad
+from torch.nn import CrossEntropyLoss
+
+from backpack.core.derivatives.basederivatives import BaseLossDerivatives
+from backpack.utils.subsampling import subsample
+
+
+class NLLLossDerivatives(BaseLossDerivatives):
+    """Partial derivative bases for NLL loss."""
+
+    def _sqrt_hessian_sampled(
+        self,
+        module: CrossEntropyLoss,
+        g_inp: Tuple[Tensor],
+        g_out: Tuple[Tensor],
+        mc_samples: int = 1,
+        subsampling: List[int] = None,
+    ) -> Tensor:
+        """Method to approximate the square root Hessian through Monte Carlo sampling.
+
+        For a given loss, either _make_distribution or compute_sampled_grads must be
+        implemented.
+
+        Optionally, it is possible to specify _checks to perform before calculating
+        the loss gradient.
+
+        For use in mean mode, _get_mean_normalization must be implemented.
+
+        Args:
+            module:
+            g_inp:
+            g_out:
```

Comment:
Add missing argument descriptions

Reply:
Fixed this in commit
@jabader97 [Implemented (some of) commented fixes](https://github.com/f-dangel/backpack/pull/252/commits/8201fc6f73bf7231f82779462f3dc46034d21c5e)